docs/tooling.rst

```diff
@@ -229,6 +230,36 @@ A new S3Compliant profile can be created using the s3compliant subcommand
   secret 's3-secret-chst2' created
   profile 's3-profile-5mmkj' created
 
+
+Kopia Repository Server creation using ``kanctl create``
```

Comment:
Should we say `Kopia Repository Server Resource creation` instead of `Kopia Repository Server creation`

Reply:
Yes !